Formula/pfetch.rb

```diff
@@ -0,0 +1,16 @@
+class Pfetch < Formula
+  desc "Pretty system information tool written in POSIX sh"
+  homepage "https://github.com/dylanaraps/pfetch/"
+  url "https://github.com/dylanaraps/pfetch/archive/0.6.0.tar.gz"
+  sha256 "d1f611e61c1f8ae55bd14f8f6054d06fcb9a2d973095367c1626842db66b3182"
+  head "https://github.com/dylanaraps/pfetch.git"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}" if (buildpath/"Makefile").exist?
```

Comment:
Why wouldn't the makefile exist?

Reply:
Because it was added after `v0.6.0` and is currently only available if you use HEAD.